src/promptflow/tests/test_configs/flows/conditional_flow_with_aggregate_bypassed/switch_case_B.py

```diff
@@ -0,0 +1,8 @@
+from promptflow import tool
+
+# The inputs section will change based on the arguments of the tool function, after you save the code
+# Adding type to arguments and return value will help the system show the types properly
+# Please update the function name/signature per need
+@tool
+def my_python_tool(input: str) -> str:
```

Comment:
When you are using two tools which are totally the same, you can reference the same file in the yaml instead of creating two scripts.